src/components/radio-button-group/RadioButtonGroup.tsx

```diff
@@ -81,11 +81,18 @@ export const RadioButtonGroup = forwardRef<HTMLDivElement, PropsWithLabels<Props
             )}
           </Label>
 
-          {helpText && <HelperText as="span">{helpText}</HelperText>}
+          {helpText && (
+            <HelperText as="span" id={`${id}-helptext`}>
+              {helpText}
+            </HelperText>
+          )}
         </RadioGroupLabelWrapper>
       )}
 
-      <RadioButtonsWrapper horizontal={alignment === RadioButtonGroupAlignment.Horizontal}>
+      <RadioButtonsWrapper
+        aria-describedby={`${id}-helptext`}
```

Comment:
We should add that only if we actually have a helptext, otherwise it's gonna target an id that doesn't exist.